Helper/Parser.php

```diff
@@ -104,6 +107,15 @@ public function parseText($text, $checkTable = true) {
 
     $text = str_replace("#", "\\#", $text);
     $text = str_replace("_", "\\_", $text);
+    $text = str_replace("$", "\\$", $text);
+    $text = str_replace("%", "\\%", $text);
+    $text = str_replace("&", "\\&", $text);
+    $text = str_replace("~", "\\~{}", $text);
+    $text = str_replace("^", "\\^{}", $text);
+    $text = str_replace("{", "\\{", $text);
+    $text = str_replace("}", "\\}", $text);
+//    $text = str_replace(">", "\\textlangle{}", $text);
+//    $text = str_replace("<", "\\textrangle{}", $text);
```

Comment:
When I test this (<,>) in my LaTeX environment, they do not need to be special commands, they actual chars work fine. 

However, when you use the in the HTML-field of the bundle, the HTML-parser cleans the < out, as it thinks they are HTML-tags. It leaves the > though (but not the first one after a <). 

Can you remove these lines from your PR?